README.md

```diff
@@ -29,6 +29,10 @@ Active development occurs in this repository, which means some contracts in it m
 - [`v2-liquidity-mining`](./pkg/liquidity-mining): contracts that compose the liquidity mining (veBAL) system.
 - [`v2-governance-scripts`](./pkg/governance-scripts): contracts that execute complex governance actions.
 
+## Pre-requisites
+
+The build & test instructions below should work out of the box with Node 14.x and Yarn 2. (Please note that it needs Node 14 specifically, and will NOT work with Node 16 or higher.)
```

Comment:
```suggestion
The build & test instructions below should work out of the box with Node 14.x. (Please note that it needs Node 14 specifically, and will NOT work with Node 16 or higher.)
```

I don't think we need the note about yarn2 as if they don't have it then running yarnv1 will automatically yarn2

Reply:
What about Yarn 3? I'm not 100% sure it works with it

Reply:
yarn v2/v3 are installed on a per-project basis. The current version of yarn being used is stored in `.yarn/releases/` and `.yarnrc.yml` points to it.